### PR TITLE
week1_fizz_buzz_phuocnt

### PR DESCRIPTION
**Pull request by at-phuocnt**

### CODE REVIEW

File: week0001/fizz_buzz_phuocnt.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+if (! function_exists('printFizzBuzz')) {
+    function printFizzBuzz($begin, $end, $requirementData) {
+        if ($begin > $end) {
+            throw new Exception('Begin number cannot greater than end number');
+        }
+        for ($i = $begin; $i <= $end; $i++) {
+            foreach ($requirementData as $numberDivided => $stringEcho) {
+                if ($i % $numberDivided == 0) {
+                    echo $stringEcho . '</br>';
```
Comment (vietlq) [inline week0001/fizz_buzz_phuocnt.php]: I guess you meant `<br/>`

File: week0001/fizz_buzz_phuocnt.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?php
+
+if (! function_exists('printFizzBuzz')) {
+    function printFizzBuzz($begin, $end, $requirementData) {
+        if ($begin > $end) {
+            throw new Exception('Begin number cannot greater than end number');
```
Comment (vietlq) [inline week0001/fizz_buzz_phuocnt.php]: `cannot be* greater`

**Comment by vietlq** [comment]
This is a bit verbose

**Comment by at-phuocnt** [comment]
@vietlq Thanks for your review :100:

**Comment by vietlq** [comment]
@at-phuocnt you are welcome! Week 0002 is coming. Please invite your friends to join the group & follow the page fb.com/Code2Pro :D

**Pull request merged**